schedule/yast/transactional_server/create_hdd_transactional_server.yaml

```diff
@@ -0,0 +1,60 @@
+name:           create_hdd_transactional_server
+description:    >
+    Installation of a Transactional Server which uses a read-only root filesystem to
+    provide atomic, automatic updates of a system without interfering with the running system.
+vars:
+    SCC_ADDONS: tsm
+    HDDSIZEGB: 40
+conditional_schedule:
+    kernel_cmd_parameters:
+        BACKEND:
+            pvm_hmc:
+            - installation/edit_optional_kernel_cmd_parameters
```

Comment:
i also think that based on the yaml specs this should have 2 space indentation. Can you check?

Reply:
Travis already checked (plus I did in local)

Reply:
but it also we could keep some standards. btw in other schedulers you also use the indentation (i believe is good to keep all of them in consistence).

Reply:
to keep those standards you might need to figure out the tool to do, perhaps it is possible with yamllint, otherwise, some will follow some will not, and it is not necessary to comment those error in each review in the future and better rely in a proper tool.

Reply:
i guess it is trivial change anyway. 
and yes, a tool will be helpful in any case.

Reply:
It is not about being trivial or not or my will to change it, we use yamllint rencently and this tool has a configuration. From the docs about [identation](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation):
```
indent-sequences: defines whether block sequences should be indented or not (when in a mapping, this indentation is not mandatory – some people perceive the - as part of the indentation). Possible values: true, false, whatever and consistent. consistent requires either all block sequences to be indented, or none to be. whatever means either indenting or not indenting individual block sequences is OK.
```
and yeah, I'm one of those people :) I think, it doesn't feel necessary to me to add yet another tab there.
So if you want to suggest a different value for [indent-sequences](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/.yamllint#L7) feel free to send a PR and discuss it there, then you could hear more preferences.

Reply:
a good formatting document is kinda important and as you already have use indentation on the other yaml i think it make sense to keep doing so in this PR. if you dont catch this in another PR it is not the end of the world but it would helpful to keep as much as common format.

Reply:
As per our discussion in RC, we will discuss this during retro and apply same indentation in all yamls, for instance.